test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -10762,6 +10785,45 @@ def forward(self, x):
         self.run_test(Module(False), x, rtol=1e-3, atol=1e-6)
         self.run_test(Module(True), x, rtol=1e-3, atol=1e-6)
 
+    @skipIfCaffe2
+    def test_clip_aten_fallback_due_exception(self):
```

Comment:
can you move these to `test/jit/test_onnx_export.py`? These tests don't use onnxruntime, so it's confusing to have them in this file.